docs/develop/auto_sdk/auto-id.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Auto ID
```

Comment:
Do we have any usage examples for AutoID functions?

Reply:
I can add a few, np 👍